docs/05-dev-tools/node-rpc/rpc-api.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 5
+title: RPC API
+sidebar_label: RPC API
+tags: [rsk, rootstock, resources, tutorials,  setup, dApps, protocol, RPC, API, ]
+description: "Remote Procedure Call (RPC) is a protocol that allows a program to execute procedures (functions) on a remote server as if they were local calls."
+---  
+ 
+Remote Procedure Call (RPC) is a protocol that allows a program to execute procedures (functions) on a remote server as if they were local calls. This eliminates the need for developers to handle low-level network communication.  
```

Comment:
Is this not supposed to reference the Rootstock RPC API?

Reply:
No, it doesnt have any reference to Rootstock RPC API? and also if we decide to add a link the page is not talking about RPC in general which is what this page is talking about.

I will add a link to line number 12